pytest_fixtures/component/subscription.py

```diff
@@ -27,9 +27,11 @@ def subscribe_satellite(clean_rhsm, module_target_sat):
         0
     ]
     if 'Successfully attached a subscription' in result.stdout:
-        module_target_sat.enable_repo(
-            f'rhel-{module_target_sat.os_version.major}-server-extras-rpms', force=True
-        )
+        # extras is not in RHEL8: https://access.redhat.com/solutions/5331391
+        if module_target_sat.os_version.major < 8:
+            module_target_sat.enable_repo(
+                f'rhel-{module_target_sat.os_version.major}-server-extras-rpms', force=True
```

Comment:
@Gauravtalreja1 I think you raise a valid point here, and I'd agree, but the scope of that is much more than the current commit scope to just limit based on version. 

I'm inclined to merge as-is, and the discussion about whether tests should enable extras via fixture inclusion (they should) can continue in an automation card or github issue. The PR addressing this will have to audit tests using this fixture to determine where updates need to be made.